### DISCUSSION TITLE
"Cannot find name" for any of react-query definition

### DISCUSSION THREAD

**Discussion by timsofteng** [TanStack/query#4547] [General]
Hello. I see massage "Cannot find name"  for any of `react-query` definition. 
I've installed react-query and `@types` for it. What do i do wrong?

**Comment by TkDodo** [comment]
please show more info than that, ideally a codesandbox reproduction.

**Comment by timsofteng** [comment]
Looks like the problem was in tsserver. Problem is gone suddenly.
